packages/maskbook/src/database/Persona/Persona.db.ts

```diff
@@ -28,11 +29,17 @@ import { CryptoKeyToJsonWebKey } from '../../utils/type-transform/CryptoKey-Json
  * A persona links to 0 or more profiles.
  * Each profile links to 0 or 1 persona.
  * @keys inline, {@link ProfileRecord.identifier}
+ *
+ * # ObjectStore `relations`:
+ * @description Store relations.
+ * @type {RelationRecord}
+ * Save the relationship between persona and profile.
+ * @keys inline {@link RelationRecord.linked  @link RelationRecord.profile}
  */
 
 const db = createDBAccessWithAsyncUpgrade<PersonaDB, Knowledge>(
-    1,
```

Comment:
The first argument is `firstVersionThatRequiresAsyncUpgrade: number`, you MUST NOT change it or the old database will break.